welcome/index.adoc

```diff
@@ -127,7 +127,7 @@ be reviewed by cluster administrators and xref:../operators/olm-adding-operators
 
 === Change cluster components
 
-- **xref:../applications/crds/crd-extending-api-with-crds.adoc[Use Custom Resource Definition (CRDs) to modify the cluster]**: Cluster features that are implemented with Operators, can be modified with CRDs. Learn to xref:../applications/crds/crd-extending-api-with-crds.adoc#crd-creating-custom-resources-definition-crd-extending-api-with-crds[create a CRD] and xref:../applications/crds/crd-managing-resources-from-crds.adoc[manage resources from CRDs].
+- **xref:../operators/crds/crd-extending-api-with-crds.adoc[Use Custom Resource Definition (CRDs) to modify the cluster]**: Cluster features that are implemented with Operators, can be modified with CRDs. Learn to xref:../operators/crds/crd-extending-api-with-crds.adoc#crd-creating-custom-resources-definition-crd-extending-api-with-crds[create a CRD] and xref:../operators/crds/crd-managing-resources-from-crds.adoc[manage resources from CRDs].
```

Comment:
@adellape I know this was already there, but would you mind fixing the anchor for the second xref? It doesn't jump to the "Creating a Custom Resource Definition" section. I think it needs an underscore instead of dash: #crd-creating-custom-resources-definition_crd-extending-api-with-crds

Also, the other xrefs don't have an anchor specified and they should, but that seems to be a problem throughout the rest of the welcome page

Reply:
Donezo.